README.md

```diff
@@ -185,6 +185,65 @@ refund = mollie_client.refunds.on(payment).create({
 })
 ```
 
+## Oauth2 ##
+
+Run the oauth2 examples.
```

Comment:
Maybe we should give some details on how to actually run this: where to get the client id/secret and how to serve your local app with a working callback (ngrok).

Reply:
Done.

Reply:
Still missing pointers on client ID and client secret. Just add a pointer to https://docs.mollie.com/oauth/getting-started I think?

Reply:
Done!